Libs/MRML/DisplayableManager/vtkMRMLAbstractDisplayableManager.cxx

```diff
@@ -105,6 +89,13 @@ class vtkMRMLAbstractDisplayableManager::vtkInternal
   static void DoMRMLInteractionNodeCallback(vtkObject* vtk_obj, unsigned long event,
                                             void* client_data, void* call_data);
 
+  /// Populates eventIdLocation with iterartor pointing to the pair containing the id or .end()
+  /// \sa UpdateInteractorStyle
+  /// \sa UpdateInteractor
+  void FindEventId(std::vector<std::pair<int, float>> idPairs,
```

Comment:
Unless there is strong reason not to, I recommend changing the function signature to
```
static std::vector<std::pair<int, float>>::iterator
FindEventId(const std::vector<std::pair<int, float>>& idPairs,
   int eventId);
```
for the following reasons:

1. static - This function does not appear to need to use the state of the class. It only uses the passed in parameters.
2. const& - changing `idPairs` to pass by const reference from pass by value reduces unnecessary copying of data.
3. return instead of out param - iterators by nature are lightweight objects good for pass by value semantics. It will also allow you to mark iterators returned by this function `const` if you don't intend to change them after setting them.